### ISSUE TITLE
Rename - Activity to Layout

### ISSUE THREAD

**Issue by yoavst** [vlonjatg/progress-activity#1]

**Comment by StephaneBg** [comment]
I think it should be ProgressRelativeLayout.

**Comment by mmazzarolo** [comment]
Yup... I think that `ProgressRelativeLayout` would fit better

**Comment by CROSP** [comment]
+++++ 
I wanted to create the same issue. The name is confusing.

**Comment by vlonjat-gashi** [comment]
Yes guys I will update the naming in the near future and also add LinearLayout (as ProgressLinearLayout) and FrameLayout (as ProgressFrameLayout) containers as well.

**Comment by yoavst** [comment]
Near future?

**Comment by prompteus** [comment]
Any updates?
